src/esp/sim/AbstractReplayRenderer.cpp

```diff
@@ -90,6 +90,10 @@ void AbstractReplayRenderer::setSensorTransformsFromKeyframe(
   return doSetSensorTransformsFromKeyframe(envIndex, prefix);
 }
 
+void AbstractReplayRenderer::render() {
+  doRender();
+}
```

Comment:
This could just call `doRender(Mn::GL::defaultFramebuffer)` instead, avoiding the need for a new `doRender()` overload altogether. 

Just FYI, a design goal I have with the private virtual `do*()` APIs is to have as little of them as possible, and all convenience handled by the public non-virtual APIs instead. A new `do*()` API means touching *seven* places in the code, while adding a convenience public API that reuses an existing `do*()` function is just one declaration and one definition.